doc/examples/features_detection/plot_scattering_classification.py

```diff
@@ -0,0 +1,259 @@
+                                                       sigma_xi=sigma_xi)
+    # Generate Training set
+    scatterings_train, u, scat_tree = scattering(X_train, Filters, m=m)
+    # Generate Testing set
+    t_scats = time.time()
+    scatterings_test, u, scat_tree = scattering(X_test, Filters, m=m)
+    time_per_image = (time.time() - t_scats) / scatterings_test.shape[0]
+    print('Scattering coefficients computed as ',
+          str(time_per_image), ' secs. per image')
+    return scatterings_train, scatterings_test
+
+
+def gethomogeneus_datast(X, y, d, n):
+    num_per_class = np.int(np.amin([n, X.shape[0]]) / d)
+    ytrain = np.reshape(y, (y.shape[0],))
+    X_out = []
+    y_out = []
+    for i_d in np.arange(d):
+        indx = np.where(ytrain.ravel() == i_d)
+        X_out.append(X[indx[0][0:num_per_class], :])
+        y_out.append(ytrain[indx[0][0:num_per_class], ])
+    X_out = np.concatenate(X_out, axis=0)
+    y_out = np.concatenate(y_out, axis=0)
+    return X_out, y_out
+
+
+##########################
+#  Classification of MNIST
+# ########################
+# Database following the results
+# presented in Bruna et al. paper.
+
+
+# Get dataset
+num_classes = 10
+px = 32  # number of pixels
+num_images = 50  # for full data set set to 50000
+n = np.amin([10000, num_images])
+
+im_train, ytrain, im_test, ytest = load_images_mnist(px=px,
+                                                     num_images=num_images)
+Xtrain, Xtest = load_scattering(im_train, im_test, px=px,
+                                J=3, L=6, m=2,
+                                sigma_phi=0.6957, sigma_xi=0.8506)
+Xtrain_1d = Xtrain.reshape((len(Xtrain), -1))  # colapse spatial components
+Xtest_1d = Xtest.reshape((len(Xtest), -1))
+# Classification using SVM with RBF Gaussian kernels
+bestC = 4.3   # parameters obtained by crossvalidation with 10000 images
+bestgamma = 10**0.1
+ns = [300, 1000, 2000, 5000, 10000, 20000, 40000, 60000]
+score_gaussian = np.zeros((len(ns), 1))
+for i, n in enumerate(ns):
+    # resample the dataset to have a uniform distribution on the classes
+    Xa, ya = gethomogeneus_datast(Xtrain_1d, ytrain, num_classes, n)
+    # define pipeline for SVM classif.
+    gs_gaussian = SVC(kernel='rbf', C=bestC, gamma=bestgamma)
+    pip_gaussian = make_pipeline(MinMaxScaler((-1, 1)),
+                                 Normalizer(), gs_gaussian)
+    pip_gaussian.fit(Xa, ya)
+    out = pip_gaussian.predict(Xtest_1d)
+    score_gaussian[i] = 1.0-accuracy_score(ytest, out)
+    # print('num training images:' + str(n) +
+    #          ' err:' + str(score_gaussian[i]*100) + '%')
+print('Warning: For exact error performance, we '
+      'need at least num_images=10000.')
+#########################
+#  Classification CIFAR10
+#########################
+# Database following the results
+# presented in Oyallon et al. paper.
+
+# Load CIFAR10
+from skimage.color.colorconv import rgb2yuv
+from keras.datasets import cifar10
+
+
+def DB_rgb2yuv(X):
```

Comment:
:) I meant explain it in a comment or better a docstring! It would be good if you could use the words that abbreviate to `DB` in that docstring, so people know why you named it that way.